ledger/sandbox/src/main/scala/platform/sandboxnext/Runner.scala

```diff
@@ -236,6 +236,9 @@ class Runner(config: SandboxConfig) extends ResourceOwner[Port] {
                   seeding = config.seeding.get,
                   managementServiceTimeout = config.managementServiceTimeout,
                   enableAppendOnlySchema = false,
+                  maxContractStateCacheSize = 0L,
```

Comment:
Can we leave a TODO comment here to augment the settings for both the append-only and mutable-cache at the same time?

Reply:
Done

Reply:
Done